src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1559,6 +1561,42 @@ public String toVariableNameString(EnchantmentOffer eo) {
 			.name("Experience Cooldown Change Reason")
 			.description("Represents a change reason of an <a href='events.html#experience cooldown change event'>experience cooldown change event</a>.")
 			.since("INSERT VERSION"));
+
+		Classes.registerClass(new ClassInfo<>(LootTable.class, "loottable")
+				.user("loot ?tables?")
```

Comment:
Not planning to do a full review but we should be registering the loottable registry not the loottable class only